js/packages/teams-ai/README.md

```diff
@@ -47,11 +49,11 @@ That's it!
 
 Run your bot (with ngrok) and sideload your manifest to test.
 
-For migrating specific features such as Message Extension and Adaptive Card capabilities, please see the [Migration Guide](../../getting-started/00.MIGRATION.md).
+For migrating specific features such as Message Extension and Adaptive Card capabilities, please see the [Migration Guide](https://github.com/microsoft/teams-ai/blob/main/getting-started/js/00.MIGRATION.md).
```

Comment:
relative paths are usually better because the links work locally as well, which is useful in particular while making updates to the docs.

Reply:
Hi @aacebo,

This README is also used for the [Teams AI Library JS NPM](https://www.npmjs.com/package/@microsoft/teams-ai?activeTab=readme) webpage. The relative links were not working on this site so after meeting with Corina, she said we needed to use global links on the README for it to work on the NPM site.

Reply:
@aacebo Totally agree - it wasn't until my meeting with Carter I realized the publish pages need to change. After discussing with @Carter425, I would like to keep relative links as much as possible, except on these pages that will be published to nuget & npmjs.